src/Components/Common/TemperatureSelector.tsx

```diff
@@ -0,0 +1,40 @@
+import { useEffect, useState } from "react";
+import { classNames } from "../../Utils/utils";
+import { SelectFormField } from "../Form/FormFields/SelectFormField";
+
+export const TemperatureSelector = (props: any) => {
+  const getTemperaturePreferance = () => {
+    if (window && window.localStorage) {
+      return localStorage.getItem("temperature");
+    }
+  };
+  const [temperature, setTemperature] = useState<string>(
+    getTemperaturePreferance() === "Fahrenheit (°F)"
+      ? "Fahrenheit (°F)"
+      : "Celsius (°C)"
+  );
+  const handleTemperature = (value: string) => {
+    setTemperature(value);
+    if (window && window.localStorage) {
+      localStorage.setItem("temperature", value);
+    }
+  };
```

Comment:
Have you thought about the use case, multiple tabs of CARE is open. What if I change the temperature from one tab, it wouldn't reflect on all other tabs right?

How about doing it with a listener on to the local storage? (like how the session expired / sign out works, it signs out from all tabs by listening to local storage)